buildbot/dependency.conf

```diff
@@ -0,0 +1,26 @@
+[VERSIONS]
+# https://github.com/intel/llvm/releases/download/2020-03/oclcpuexp-2020.10.4.0.15_rel.tar.gz
+ocl_cpu_rt_ver=2020.10.4.0.15
+# https://github.com/intel/llvm/releases/download/2020-03/win-oclcpuexp-2020.10.4.0.15_rel.zip
+ocl_cpu_rt_ver_win=2020.10.4.0.15
+# https://github.com/intel/compute-runtime/releases/tag/20.12.16259
+ocl_gpu_rt_ver=l0-20.12.16259
+# https://downloadcenter.intel.com/download/29557/Intel-Graphics-Windows-10-DCH-Drivers
+ocl_gpu_rt_ver_win=ci-neo-015900
```

Comment:
Please double check that this is the right version. Approving change anyway.

Reply:
This version is accurate